bbb-learning-dashboard/Readme.md

```
@@ -53,16 +59,25 @@ Behavior:
 
 ## Standalone mode
 
-Built with `REACT_APP_STANDALONE_MODE=true`. Use this when the dashboard is served as a self-contained static application, decoupled from a live BBB server.
+Built with `VITE_STANDALONE_MODE=true`. Use this when the dashboard is served as a self-contained static application, decoupled from a live BBB server.
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)